pandas/tests/groupby/test_groupby.py

```diff
@@ -3843,6 +3843,128 @@ def test_cummin_cummax(self):
         expected = pd.Series([1, 2, 1], name='b')
         tm.assert_series_equal(result, expected)
 
+    def test_is_increasing_is_decreasing(self):
+        # GH 17015
+
```

Comment:
we don't really need all this, as ``is_montonic`` and friends are already well tested in ``Index`` (where they are actually implemented), but no big deal.

Reply:
figured as much but wasn't sure whether to test here and risk duplication or not test here and have "untested" code.